player/lua/console.lua

```diff
@@ -567,8 +578,9 @@ local function update()
         -- Even with bottom-left anchoring,
         -- libass/ass_render.c:ass_render_event() subtracts --osd-margin-x from
         -- the maximum text width twice.
-        local width_max = math.floor((screenx - marginx - marginx * 2 / scale_factor())
-                                     / opts.font_size * get_font_hw_ratio())
+        local width_max = math.floor(
+            (screenx - marginx - mp.get_property_native('osd-margin-x') * 2 / scale_factor())
```

Comment:
Why `mp.get_property_native('osd-margin-x')`?

Reply:
Because libass still applies that to determine where to wrap. Though it is currently not accurate because as you noticed we scale margin-x incorrectly because it should scale with PlayResX.

Reply:
Is this bug in libass? We position console with absolute coordinates. So why margix-x from the style matters?

Reply:
We anchor console to the left, why wouldn't the right margin matter? It probably shouldn't apply the margin twice if absolutely positioned but I have no idea if that should be changed in libass.

Reply:
Ok, I though there is no right margin, because it generally doesn't work from what I remember, but seems like an unrelated problem.